config/initializers/assets.rb

```diff
@@ -4,7 +4,7 @@
 Rails.application.config.assets.version = '1.0'
 
 Rails.application.config.assets.precompile += %w( gov.uk_logotype_crown.svg )
-Rails.application.config.assets.precompile += %w( elements-page-ie6.css elements-page-ie7.css elements-page-ie8.css )
+Rails.application.config.assets.precompile += %w( ie8.css.scss )
```

Comment:
If tested and works ignore this comment, but seem to remember you need the compiled name here eg `ie8.css` - see e.g. this [SO](http://stackoverflow.com/questions/7367864/precompiling-a-scss-manifest-file-using-rails-3-1s-asset-pipeline) - but like I say, might not be an issue if works as is.